cmd/zarb/main.go

```diff
@@ -21,9 +22,13 @@ func zarb() *cli.Cli {
 		k.Command("verify", "Verify a signature", key.Verify())
 		k.Command("change-auth", "Change the passphrase of a keyfile", key.ChangeAuth())
 	})
-	app.Command("tx", "Create raw transaction", func(k *cli.Cmd) {
-		k.Command("bond", "Generate raw bond transaction", tx.BondTx())
-		k.Command("send", "Generate raw send transaction", tx.SendTx())
+	app.Command("tx", "Create, Sing, Publish transactions", func(k *cli.Cmd) {
+		k.Command("bond", "Send signed Bond transaction to network", tx.BondTx())
+		k.Command("send", "Send signed Send transaction to network", tx.SendTx())
+	})
+	app.Command("payload", "Create raw transaction", func(k *cli.Cmd) {
```

Comment:
AS I checked your code, I am convinced that maybe we don't need the old style.

Reply:
we might need them for smart-contracts implementations so i suggest keeping them

Reply:
This is redundant to me now. With the new commands, we can still make transactions and sign them off line. 
Please consider that users might raise issues about them. Let archive them now.